tests/test_reorder_chains.py

```
@@ -0,0 +1,410 @@
+        _make_telemetry_db(
+            self.db_path,
+            [
+                {"provider": "bad", "model": "m-bad", "status": "error", "tps": 1.0, "ttft_ms": 5000.0}
+                for _ in range(10)
+            ]
+            + [
+                {"provider": "good", "model": "m-good", "status": "success", "tps": 200.0, "ttft_ms": 100.0}
+                for _ in range(10)
+            ],
+        )
+        total, reordered, log = reorder_config(
```

Ruff (0.15.17)

[warning] Unpacked variable `log` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)